src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs

```diff
@@ -168,6 +169,12 @@ public string TizenExpansionPackageName
         /// <since_tizen> 3 </since_tizen>
         public IEnumerable<string> Privileges { get { return _privileges; } }
 
+        /// <summary>
+        /// First installed time of the package.
```

Comment:
Maybe:
```suggestion
        /// The timestamp of the applications first installation.
```
Or "when the application was installed for the first time"

Reply:
I take third suggestion and change application to package.
```
The timestamp the package was installed for the first time
```